fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadata.java

```diff
@@ -179,6 +180,10 @@ default void dropTable(DropTableStmt stmt) throws DdlException {
         throw new StarRocksConnectorException("This connector doesn't support dropping tables");
     }
 
+    default void finishSink(String dbName, String table, List<TSinkCommitInfo> commitInfos) {
```

Comment:
Why call it `finishSink` instead of `commit`?

Reply:
`commit` is too common, I think `finish_sink` is more representative what I want to do.